bin/tfenv

```diff
@@ -60,8 +60,15 @@ done;
 
 declare arg="${1:-""}";
 
-log 'debug' "Setting TFENV_DIR to ${PWD}";
-export TFENV_DIR="${PWD}"
+for _arg in ${@:1}; do
+    if [[ "$_arg" == -chdir=* ]]; then
+        export TFENV_DIR="${PWD}/${_arg#-chdir=}";
+        break;
+    else
+        export TFENV_DIR="${PWD}";
+    fi;
+done;
+log 'debug' "Setting TFENV_DIR to ${TFENV_DIR}";
```

Comment:
Is it possible to add a test case for this? I don't use the chdir parameter, it would be good to have some repeatable confidence in this option.

Reply:
👍🏻 I'll have a look.

Reply:
I've added a test, but (as it shows) this seems not to work any more (I have been using it continuously since I opened the PR, it definitely _did_) on latest master; looking into it, just FYI that it's not mergeable at present.

Reply:
Ok, working again, but the test seems a bit flakey - not sure if it's just my environment messed up somehow, it's ok in manual testing, could you approve it to test in CI?